src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5056,23 +4989,9 @@ static void Main()
     }
 }
 ";
+            // This is permitted, as long as it doesn't use an expression lvalue in the deconstruction.
```

Comment:
Julien is working on semantic model support now. I'll do a runtime test.

---
In reply to: [90168304](https://github.com/dotnet/roslyn/pull/15548#discussion_r90168304) [](ancestors = 90168304)